e2e/app.e2e-spec.ts

```diff
@@ -0,0 +1,14 @@
+import { AppPage } from './app.po';
```

Comment:
does it make much sense to keep e2e test configurations and the stub tests in this repo?

Reply:
... I'd either completely remove e2e testing stuff or just fix this one and make it very basic.